.github/workflows/node.js.yml

```diff
@@ -1,25 +0,0 @@
-# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
```

Comment:
Why is this file getting deleted?

Reply:
Ah, I see why now -- let's not rename this file, please!

Reply:
Do you have an alternate suggestion? `node.js.yml` is an odd default that comes from the Actions UI wizard

Reply:
Let's think about a new name at a later time.  The filename doesn't make much of a difference, so I'd rather keep the filename the same for now to retain the Git history!

Reply:
There doesn't really seem to be any important history to retain for the file https://github.com/mdn/browser-compat-data/commits/main/.github/workflows/node.js.yml but I've reverted the name to avoid this getting stalled with bikeshedding the name